WordPressComKit/MeService.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import Alamofire
+
+public class MeService {
+    public init() {}
+    
+    public func fetchMe(completion: (Me?, NSError?) -> Void) {
```

Comment:
I definitely don't like the `completion` pattern here. At compile time, a completion block could be called with `(nil, nil)` or with `(me, error)`, leading to unexpected scenarios.

Some alternatives:

- Two blocks: `success`/`failure`, old style.
- One `completion` block with a [Result](https://github.com/antitypical/Result) type.
- Build on Alamofire chainable requests and be able to do `service.fetchMe().success(_).failure(_)`